lib/WebpackOptionsDefaulter.js

```diff
@@ -73,6 +73,11 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			{
 				test: /\.wasm$/i,
 				type: "webassembly/experimental"
+			},
+			{
+				test: /\.wast$/i,
+				loader: "@webassemblyjs/wast-loader",
```

Comment:
I don't know if this is wanted but I think it's a common use case to enable it by default?

Reply:
Using loaders by default comes with a lot of problems. I would keep it like other loaders.

Reply:
For the test suite move this into `test/TestCases.template.js`